src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,18 @@
+package duke.task;
+
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String outputFormat() {
+        return "T" + super.outputFormat();
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
```

Comment:
Should include header comments for all public methods? But maybe not for overrides I'm not too sure.